src/groupeddataframe/complextransforms.jl

```diff
@@ -261,7 +261,7 @@ function _combine_rows_with_first!((firstrow,)::Ref{Any},
     # Create up to one task per thread
     # This has lower overhead than creating one task per group,
     # but is optimal only if operations take roughly the same time for all groups
-    if VERSION >= v"1.4" && isthreadsafe(outcols, incols)
+    if VERSION >= v"1.4" && MULTITHREADING[] && isthreadsafe(outcols, incols)
```

Comment:
This pattern has a problem that we need to resolve before merging.

The problem is that after this check abut before you call `@spawn_or_async` (or `@spawn_or_run`) the value of `MULTITHREADING[]` might change.

What I propose is to set a local variable in this `if` part that captures if multi-threading should be used and pass this flag to `@spawn_or_async`.

You know the code better, so maybe this will not cause problems in any place, but it seems it would be safer to do this this way.

Reply:
Good point. I don't think it can be a problem though. In both cases we create a task, the only difference is whether we set `t.sticky` to `true` or `false`.